data/projects/all.json

```diff
@@ -235,9 +235,52 @@
             "sonatype_lifecycle": false
         },
         "tags": [
-            "nexus",
-            "repository",
             "blobstore"
         ]
+    },
+    {
+        "name": "API Clients",
```

Comment:
```suggestion
        "name": "Nexus Repo API Clients",
```

Reply:
I have issue with this - not aligned to official brand names - should be `Sonatype Nexus Repository Manager API Clients` - which is too long...

Reply:
"API Clients (Repo)"